src/Utilities/Helpers.js

```diff
@@ -602,6 +602,25 @@ export const convertIsoToDate = (isoDate) => {
         `-${dateObject.getDate().toString().padStart(2, '0')}`;
 };
 
+// 2023-03-05 -> 05 Mar 2023
+// 2023-03-22T20:00:00-04:00 (ISO format) -> 23 Mar 2023
+export const templateDateFormat = (dateString) => {
```

Comment:
One unit test would make this even more robust 👍

Reply:
Very good suggestion! I just added some :grinning: